src/core/qgis.cpp

```diff
@@ -83,6 +83,7 @@ const double Qgis::DEFAULT_HIGHLIGHT_MIN_WIDTH_MM = 1.0;
 
 const double Qgis::SCALE_PRECISION = 0.9999999999;
 
+const double Qgis::DEFAULT_Z_COORDINATE = 0.0;
```

Comment:
I second @wonder-sk, no need to introduce Qgis::SCALE_PRECISION, it's only used in one place and doesn't clarify the code.